README.md

```diff
@@ -12,26 +12,57 @@
 
 ## Purpose
 
-It is an Open Source experiment aiming to build a next generation Open Source CMS solution merging together many new approaches:
+Citation is a new way of building websites. It brings to users as well as developers the ability to fully configure a website by specifying the routing and organizing all the components of each page.
+
+It's able to manage your website structure as well as all your contents through an easy to use administration interface which store every data in a Git repository which will you full control on your data: historic, branches, merges...
```

Comment:
manage your website structure and your contents

Je verrais bien un découpage de la fin de la phrase car les deux which imbriqués font mal.
This repository will handle full control on your..